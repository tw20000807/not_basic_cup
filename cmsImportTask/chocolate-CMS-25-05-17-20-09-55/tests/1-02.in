dmv7A5gYlPmjptJm7mx15JsQPSMiOxJQ
2
500
149 79
23 8
152 116
494 124
149 43
81 31
380 251
31 27
369 358
422 320
220 30
347 49
216 3
422 335
494 249
165 26
308 69
315 277
257 255
480 69
231 57
42 33
388 193
86 34
200 109
194 158
392 25
77 49
318 76
236 231
213 31
40 30
490 132
167 115
439 361
451 218
388 216
452 351
327 156
122 70
392 173
253 139
283 5
38 30
364 152
253 24
39 33
89 83
57 8
270 256
146 80
366 48
172 65
87 64
41 37
222 103
272 145
104 5
389 34
351 234
83 61
262 101
489 1
103 88
81 80
251 188
373 78
132 111
441 126
2 2
218 119
41 29
499 354
107 62
372 250
57 22
193 193
229 151
405 153
472 266
78 21
60 38
492 488
310 207
139 4
301 268
62 30
271 22
274 46
202 60
380 98
56 30
97 50
131 129
501 338
196 82
461 82
491 139
215 194
204 83
177 8
452 430
431 319
455 1
218 2
380 353
383 231
27 19
502 18
345 107
25 11
297 46
24 22
41 27
399 89
115 43
231 205
323 275
182 95
10 2
369 166
420 229
451 262
161 72
408 374
351 193
283 24
402 70
492 202
439 179
103 64
402 83
117 45
289 20
406 401
442 273
199 8
176 92
484 34
114 13
110 49
187 93
373 22
365 355
441 323
291 92
484 449
205 171
83 9
9 4
126 68
223 175
45 11
418 140
257 125
245 100
323 143
480 140
457 48
498 369
53 11
79 44
7 1
103 26
152 51
193 56
479 192
305 207
281 202
425 331
447 56
263 67
54 48
244 32
391 12
427 237
103 15
233 148
340 186
405 163
406 110
327 301
367 209
164 42
368 266
56 49
286 106
4 2
369 105
405 345
141 17
183 81
276 237
397 96
151 42
435 377
506 75
459 399
336 110
103 3
502 396
66 62
436 390
135 47
312 67
152 65
210 69
367 72
215 31
133 71
148 69
324 85
77 77
191 170
350 135
223 185
390 171
107 78
433 169
461 63
393 105
87 49
259 190
135 75
437 101
251 186
277 80
411 330
326 255
31 8
111 62
508 233
377 323
26 22
38 21
416 384
23 8
228 155
244 238
2 2
18 3
303 86
216 148
165 78
327 312
303 256
301 15
26 6
233 150
207 38
377 299
264 157
31 9
491 112
364 349
508 249
46 40
371 237
323 316
194 191
375 63
247 2
445 230
230 99
198 102
293 49
258 145
135 65
304 299
6 5
19 14
55 26
293 67
18 1
197 88
103 49
458 301
368 321
345 55
407 147
464 200
37 15
488 452
124 35
154 109
383 332
380 224
42 24
492 23
477 319
29 3
197 18
467 219
268 19
128 25
321 11
345 306
487 479
146 9
498 23
346 222
433 346
145 130
325 111
403 54
134 55
481 210
432 104
471 379
88 77
468 319
185 88
170 15
321 310
116 46
44 3
342 211
51 44
263 51
212 184
148 51
310 46
37 11
129 71
213 72
146 122
50 11
380 14
281 4
502 378
461 300
164 17
393 88
235 101
27 16
139 88
37 5
408 389
357 136
174 134
161 102
346 197
110 38
417 38
449 94
460 379
330 143
270 66
102 94
336 79
279 13
250 111
350 200
200 80
380 99
20 11
352 175
156 78
475 320
498 161
70 27
89 25
76 21
196 158
331 145
200 70
252 94
78 31
152 7
102 5
172 65
164 142
472 109
75 34
197 34
334 258
312 206
63 62
414 299
403 122
424 50
371 111
349 323
273 180
506 470
149 62
80 67
409 212
85 79
381 229
410 209
178 127
109 46
427 303
211 140
192 108
338 326
168 14
366 165
444 8
22 14
66 36
227 35
511 44
453 197
156 21
381 124
165 8
371 277
101 56
478 255
482 178
222 115
294 192
125 1
201 71
247 3
50 50
354 150
273 127
473 454
392 261
379 51
472 76
471 165
476 177
457 216
339 135
328 28
161 67
199 118
334 315
28 5
468 271
64 55
392 336
334 274
194 63
45 39
391 29
172 37
386 4
96 71
96 76
182 57
230 62
324 119
43 11
466 313
423 80
493 366
282 62
116 7
276 78
257 108
344 344
221 151
497 94
298 281
67 19
251 113
375 89
272 121
296 59
343 333
472 276
299 82
337 104
241 43
305 206
237 76
124 15
100 30
209 33
274 108
393 334
64 28
1 1
60 8
346 108
270 26
471 145
98 66
217 5
244 210
149 126
130 28
118 65
370 84
168 117
240 12
508 497
439 207
290 209
81 10
406 19
44 3
323 31
472 78
70 31
