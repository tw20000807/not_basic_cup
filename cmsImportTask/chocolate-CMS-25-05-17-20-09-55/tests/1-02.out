UUHvViSbOcm1f20T0qa9HbSlGWAR2d3H
500
OK
7
OK
4
OK
7
OK
7
OK
6
OK
5
OK
8
OK
5
OK
9
OK
9
OK
5
OK
6
OK
2
OK
9
OK
8
OK
5
OK
7
OK
9
OK
8
OK
7
OK
6
OK
6
OK
8
OK
6
OK
7
OK
8
OK
5
OK
6
OK
7
OK
8
OK
5
OK
5
OK
8
OK
7
OK
9
OK
8
OK
8
OK
9
OK
8
OK
7
OK
8
OK
8
OK
3
OK
5
OK
8
OK
5
OK
6
OK
7
OK
4
OK
9
OK
7
OK
6
OK
7
OK
7
OK
6
OK
7
OK
8
OK
3
OK
6
OK
8
OK
6
OK
7
OK
1
OK
7
OK
1
OK
8
OK
7
OK
7
OK
7
OK
0
OK
7
OK
5
OK
9
OK
6
OK
8
OK
5
OK
0
OK
8
OK
8
OK
9
OK
5
OK
6
OK
9
OK
8
OK
3
OK
9
OK
5
OK
5
OK
6
OK
6
OK
7
OK
5
OK
6
OK
2
OK
9
OK
7
OK
7
OK
8
OK
8
OK
7
OK
4
OK
9
OK
9
OK
1
OK
2
OK
9
OK
8
OK
5
OK
5
OK
7
OK
4
OK
6
OK
5
OK
5
OK
7
OK
6
OK
8
OK
9
OK
7
OK
2
OK
8
OK
8
OK
9
OK
7
OK
9
OK
8
OK
5
OK
7
OK
8
OK
8
OK
7
OK
7
OK
6
OK
5
OK
9
OK
9
OK
4
OK
7
OK
6
OK
4
OK
6
OK
7
OK
5
OK
9
OK
9
OK
7
OK
9
OK
8
OK
4
OK
3
OK
7
OK
8
OK
4
OK
8
OK
7
OK
7
OK
8
OK
8
OK
6
OK
9
OK
4
OK
6
OK
1
OK
5
OK
6
OK
6
OK
8
OK
8
OK
8
OK
9
OK
6
OK
7
OK
6
OK
6
OK
4
OK
8
OK
4
OK
8
OK
8
OK
8
OK
7
OK
9
OK
8
OK
6
OK
9
OK
6
OK
7
OK
2
OK
7
OK
9
OK
5
OK
7
OK
8
OK
7
OK
6
OK
9
OK
7
OK
9
OK
7
OK
2
OK
9
OK
6
OK
9
OK
6
OK
7
OK
7
OK
7
OK
7
OK
5
OK
7
OK
7
OK
7
OK
0
OK
8
OK
8
OK
8
OK
8
OK
7
OK
8
OK
6
OK
7
OK
6
OK
8
OK
7
OK
7
OK
8
OK
7
OK
9
OK
8
OK
4
OK
6
OK
8
OK
9
OK
5
OK
5
OK
9
OK
4
OK
8
OK
8
OK
0
OK
2
OK
7
OK
8
OK
7
OK
9
OK
9
OK
4
OK
3
OK
8
OK
6
OK
9
OK
8
OK
4
OK
7
OK
9
OK
8
OK
6
OK
8
OK
9
OK
8
OK
6
OK
2
OK
8
OK
7
OK
7
OK
6
OK
8
OK
7
OK
9
OK
3
OK
4
OK
5
OK
7
OK
1
OK
7
OK
6
OK
9
OK
9
OK
6
OK
8
OK
8
OK
4
OK
9
OK
6
OK
7
OK
9
OK
8
OK
5
OK
5
OK
9
OK
2
OK
5
OK
8
OK
5
OK
5
OK
4
OK
9
OK
9
OK
4
OK
5
OK
8
OK
9
OK
8
OK
7
OK
6
OK
6
OK
8
OK
7
OK
9
OK
7
OK
9
OK
7
OK
4
OK
9
OK
6
OK
2
OK
8
OK
6
OK
6
OK
8
OK
6
OK
6
OK
4
OK
7
OK
7
OK
7
OK
4
OK
4
OK
3
OK
9
OK
9
OK
5
OK
7
OK
7
OK
5
OK
7
OK
3
OK
9
OK
8
OK
8
OK
7
OK
8
OK
6
OK
6
OK
7
OK
9
OK
8
OK
7
OK
7
OK
7
OK
4
OK
7
OK
8
OK
7
OK
7
OK
4
OK
8
OK
7
OK
9
OK
8
OK
5
OK
5
OK
5
OK
8
OK
8
OK
7
OK
7
OK
5
OK
3
OK
3
OK
7
OK
8
OK
7
OK
6
OK
6
OK
9
OK
8
OK
6
OK
9
OK
7
OK
6
OK
7
OK
9
OK
8
OK
9
OK
6
OK
7
OK
8
OK
7
OK
8
OK
8
OK
7
OK
6
OK
9
OK
8
OK
7
OK
9
OK
4
OK
8
OK
4
OK
4
OK
6
OK
6
OK
6
OK
8
OK
5
OK
7
OK
4
OK
9
OK
6
OK
8
OK
8
OK
7
OK
8
OK
1
OK
7
OK
2
OK
0
OK
8
OK
7
OK
9
OK
9
OK
6
OK
7
OK
8
OK
8
OK
8
OK
8
OK
5
OK
7
OK
7
OK
9
OK
3
OK
9
OK
6
OK
9
OK
9
OK
6
OK
6
OK
5
OK
6
OK
3
OK
7
OK
7
OK
6
OK
6
OK
7
OK
4
OK
9
OK
7
OK
9
OK
6
OK
3
OK
7
OK
7
OK
0
OK
8
OK
7
OK
9
OK
5
OK
7
OK
7
OK
7
OK
6
OK
9
OK
9
OK
7
OK
7
OK
6
OK
8
OK
7
OK
4
OK
5
OK
6
OK
7
OK
9
OK
5
OK
1
OK
4
OK
7
OK
5
OK
8
OK
7
OK
3
OK
8
OK
7
OK
5
OK
7
OK
7
OK
7
OK
4
OK
9
OK
8
OK
8
OK
4
OK
5
OK
2
OK
5
OK
7
OK
5
